tools-image/build-arm-image.sh

```diff
@@ -473,7 +494,11 @@ sync
 sleep 5
 sync
 
-kpartx -dv $DRIVE || true
+if [ "$model" == 'rpi4' ]; then
+  kpartx -dvg $DRIVE
```

Comment:
```suggestion
  kpartx -dvg $DRIVE || true
```

maybe?

Reply:
oki then no need

Reply:
I dunno, tdo you think its necessary? Does this usually fail?

I want to clean this up in order to be able to runs it several times locally with no failures and it seems to work as expected, but of course if it fails somewhere else...